broker/src/main/java/io/camunda/zeebe/broker/partitioning/topology/StaticPartitionDistributionResolver.java

```diff
@@ -26,10 +26,10 @@
 /**
  * Manage how partitions are distributed among the brokers.
  *
- * <p>Note: Do not confuse it with the {@link TopologyManager}. {@link ClusterTopologyManager}
- * doesn't keep track of the current leader and followers.
+ * <p>Note: Do not confuse it with the {@link TopologyManager}. {@link
```

Comment:
🙃️ the new name means it's not really likely to confuse it with the TopologyManager anymore